patches/@types/dd-trace+0.5.1.patch

```diff
@@ -0,0 +1,141 @@
+patch-package
```

Comment:
This file is because the DT update hasn’t been published yet to npm. Let’s double-check if it’s available by the time you’re reviewing this and, if so, I can update that.

Reply:
@alloy - Latest is 0.6.0 now so this can be removed